pvm.py

```diff
@@ -27,9 +29,10 @@ def _init_logger():
 
 _init_logger()
 _logger = logging.getLogger("PVM")
-_logger.info("Logging system initilized in %s", os.getcwd())
+_logger.info("Logging system initiated in %s", LOG_PATH)
 
 # Place your videos in this folder for autostart
+LOG_PATH = "/home/pi/PVM/log/"
 PEFIX_PATH = "/home/pi/Videos/"
```

Comment:
same thing with this path. just seeing it now

Reply:
Same here, should be fixed with #8 .